repo-template/.github/workflows/lint.yml

```
@@ -1,5 +1,13 @@
 name: Lint
-on: [push, pull_request_target]
+on:
+    push:
+      paths:
+        - '*.lua'
+        - '.github/workflows/lint.yml'
+    pull_request_target:
+      paths:
+        - '*.lua'
+        - '.github/workflows/lint.yml'
```

```suggestion
  push:
    paths:
      - '*.lua'
      - '.github/workflows/lint.yml'
  pull_request_target:
    paths:
      - '*.lua'
      - '.github/workflows/lint.yml'
```
Not sure how much this indentation affects YAML but we should keep it the same.